crates/typst-cli/src/args.rs

```diff
@@ -241,10 +241,14 @@ pub struct CompileArgs {
     #[arg(long = "pages", value_delimiter = ',')]
     pub pages: Option<Vec<Pages>>,
 
-    /// One (or multiple comma-separated) PDF standards that Typst will enforce
+    /// The version of the produced PDF.
+    #[arg(long = "pdf-version")]
+    pub pdf_version: Option<PdfVersion>,
+
+    /// A PDF standard that Typst will enforce
     /// conformance with.
-    #[arg(long = "pdf-standard", value_delimiter = ',')]
-    pub pdf_standard: Vec<PdfStandard>,
+    #[arg(long = "pdf-standard")]
+    pub pdf_standard: Option<PdfStandard>,
```

Comment:
Is this temporary or do you intend to keep this change? In principle, it was intentional that
- `--pdf-standard` was a list (e.g. you could conform to `a2-a`, `ua-1`, and `x-4` at the same time)
- `--pdf-version` was handled as just another standard

Reply:
I would have intended to keep this change, because 1) it's not supported anyway (and wasn't supported before) and 2) I don't think it's super common to target multiple standards? And I think having version and sub-standard separately makes sense, because they can be combined artbirarily, and it doesn't make sense to target more than 1 PDF version.

Reply:
I added back the option to specify multiple PDF standards, but will still keep version selection separately for now.